Automattic/portkey-android
# Story Frame Selection : First frame selected for editing

Fixes #337

Builds on top of #386 
WPAndroid PR https://github.com/wordpress-mobile/WordPress-Android/pull/12297

## Findings
Currently, we are always using the last frame that's added as the default.

## Solution
To utilize the list of URIs that are added to determine the default selection. The default selection should always be the index of the first item of the URI list. 

## Testing
1. Create a new story. 
2. Add more than one image. 
3. Notice that the first image is the one that's selected by default. 

## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
